nio_cli/cli.py

```diff
@@ -2,21 +2,23 @@
 nio
 
 Usage:
-  nio new <project-name> [(--template <template> | -t <template>)] [(--pubkeeper-hostname=HOSTAME --pubkeeper-token=TOKEN)]
+  nio new <project-name> [(--template <template> | -t <template>)] [(--pubkeeper-hostname=HOSTAME --pubkeeper-token=TOKEN)] [--username=] [--password=]
   nio newblock <block-name>
   nio [options] add <block-repo>... [(--upgrade | -u)]
-  nio [options] (list | ls) services
-  nio [options] (list | ls) blocks
-  nio [options] (command | co) <command-name> [<service-name>] [<block-name>] [--args=<args>]
-  nio [options] (config | cfg) [project]
-  nio [options] (config | cfg) services <service-name>
-  nio [options] (config | cfg) blocks <block-name>
-  nio [options] clone services <service-name> <new-name>
+  nio [options] (list | ls) services [--username=] [--password=]
+  nio [options] (list | ls) blocks [--username=] [--password=]
+  nio [options] (command | co) <command-name> [<service-name>] [<block-name>] [--args=<args>] [--username=] [--password=]
+  nio [options] (config | cfg) [project] [--username=] [--password=]
+  nio [options] (config | cfg) services <service-name> [--username=] [--password=]
+  nio [options] (config | cfg) blocks <block-name> [--username=] [--password=]
+  nio [options] clone services <service-name> <new-name> [--username=] [--password=]
   nio [options] buildspec <repo-name>
   nio [options] buildreadme
   nio [options] buildrelease <repo-name>
   nio [options] blockcheck
-  nio [options] shutdown
+  nio [options] shutdown [--username=] [--password=]
+  nio [options] add_user [<project-name>] [--username=] [--password=]
```

Comment:
Just to be clear, `--username` and `--password` are different when calling `add_user` or `remove_user`, so I think those should stay. In other requests the username/pass define how you want to connect to your nio instance, in this one they define the user that is being added.